### PR TITLE
ENG-478 Transact xcm functions

### PR DESCRIPTION
**Pull request by lreesby**

### CODE REVIEW

File: pallets/xcmp-handler/src/lib.rs [hunk 1]
```diff
@@ -311,5 +343,70 @@ pub mod pallet {
...
+
+		/// Transact XCM instructions on local chain
+		///
+		pub fn transact_in_local_chain(
+			internal_instructions: xcm::v2::Xcm<<T as pallet::Config>::Call>,
+		) -> Result<(), DispatchError> {
+			let local_sovereign_account =
+				MultiLocation::new(1, X1(Parachain(T::SelfParaId::get().into())));
+			let weight = T::Weigher::weight(&mut internal_instructions.clone().into())
+				.map_or(Weight::max_value(), |w| T::BaseXcmWeight::get().saturating_add(w));
```
Comment (rhuttman) [inline pallets/xcmp-handler/src/lib.rs:355]: how does this base weight work? do we need it in the weight for the target chain instructions?

**Review by rhuttman — COMMENTED** [review]
just 1 Q

File: pallets/xcmp-handler/src/lib.rs (hunk 1 above)
Comment (lreesby) [reply pallets/xcmp-handler/src/lib.rs:355]: It is set here: https://github.com/OAK-Foundation/OAK-blockchain/blob/69b140aa21903f6f2a887a164a9e72899874c5ee/runtime/turing/src/xcm_config.rs#L404

Comment (lreesby) [reply pallets/xcmp-handler/src/lib.rs]: removed after chatting.

**Review by rhuttman — APPROVED** [review]

**Pull request merged**